src/components/admin/Api/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Divider, Grid, SxProps, Theme, Typography } from '@mui/material';
+import { Auth } from '@supabase/ui';
+import { authenticatedRoutes } from 'app/routes';
+import AdminTabs from 'components/admin/Tabs';
+import SingleLineCode from 'components/content/SingleLineCode';
+import usePageTitle from 'hooks/usePageTitle';
+import { FormattedMessage } from 'react-intl';
+import RefreshToken from './RefreshToken';
+
+const boxStyling: SxProps<Theme> = {
+    padding: 2,
+};
+
+function AdminApi() {
+    usePageTitle({
+        header: authenticatedRoutes.admin.api.title,
+        headerLink:
+            'https://docs.estuary.dev/reference/authentication/#authenticating-flow-using-the-cli',
+    });
+
+    const { session } = Auth.useUser();
+
+    return (
+        <>
+            <AdminTabs />
+
+            <Grid container spacing={{ xs: 3, md: 2 }} sx={{ p: 2 }}>
+                <Grid item xs={12} md={9}>
+                    <Typography variant="h6" sx={{ mb: 0.5 }}>
+                        <FormattedMessage id="admin.cli_api.header" />
+                    </Typography>
+
+                    <Typography>
+                        <FormattedMessage id="admin.cli_api.message" />
+                    </Typography>
+                </Grid>
+            </Grid>
+
+            <Divider sx={{ mt: 1, mb: 2 }} />
+
+            <RefreshToken />
+
+            <Box sx={boxStyling}>
+                <Typography
+                    sx={{
+                        mb: 0.5,
+                        fontSize: 18,
+                        fontWeight: '400',
+                    }}
+                >
+                    <FormattedMessage id="admin.cli_api.accessToken" />
+                </Typography>
+
+                <Typography sx={{ mb: 3 }}>
+                    <FormattedMessage id="admin.cli_api.accessToken.message" />
+                </Typography>
+
+                {/* TODO (defect): Display an error in the event the access token does not exist. */}
+                <SingleLineCode value={session?.access_token ?? ''} />
+            </Box>
```

Comment:
This should be broken out to a stand alone component just to keep it in sync with how the `Refresh` stuff is handled